.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -161,11 +161,18 @@ public virtual ParseResult Parse(ExtractResult extResult)
             // Add "offset" and "relativeTo" for ordinal
             if (!string.IsNullOrEmpty(ret.Type) && ret.Type.Contains(Constants.MODEL_ORDINAL))
             {
-                if ((this.Config.Options & NumberOptions.SuppressExtendedTypes) == 0 && Config.RelativeReferenceOffsetMap.ContainsKey(extResult.Text) &&
-                    Config.RelativeReferenceRelativeToMap.ContainsKey(extResult.Text))
+                if ((this.Config.Options & NumberOptions.SuppressExtendedTypes) == 0 && ret.Metadata.IsOrdinalRelative)

```

Comment:
Do we really need to set extra metadata to use here? Can't this be represented somewhere else in the extractor?
If not, that's fine. But can't the final type be set here too?